packages/vue/src/utils/index.js

```diff
@@ -204,3 +204,8 @@ export const debounce = (method, delay) => {
 		method();
 	}, delay);
 };
+export function decodeHtml(html) {
```

Comment:
@mohdashraf010897 `document` would be undefined in SSR. Handle it.

Reply:
Can use something like this: https://stackoverflow.com/a/7394814

```
function parseHtmlEntities(str) {
    return str.replace(/&#([0-9]{1,3});/gi, function(match, numStr) {
        var num = parseInt(numStr, 10); // read num as normal number
        return String.fromCharCode(num);
    });
}
```